docs/metrics/gpt4-prompt-eng.md

```diff
@@ -0,0 +1,88 @@
+# GPT4 Prompt Engineering
```

Comment:
regarding image - sick! do we have a dark mode version, and can we use brand fonts/colours?
May need to ask Phoebe.